site/content/docs/howto/configure-supervisor-with-auth0.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Configure the Pinniped Supervisor to use Auth0 as an OIDC provider
+description: Set up the Pinniped Supervisor to use Auth0 login.
+cascade:
+  layout: docs
+menu:
+  docs:
+    name: Configure Supervisor With Auth0 OIDC
+    weight: 80
+    parent: howtos
+---
+The Supervisor is an [OpenID Connect (OIDC)](https://openid.net/connect/) issuer that supports connecting a single
+"upstream" identity provider to many "downstream" cluster clients.
+
+This guide shows you how to configure the Supervisor so that users can authenticate to their Kubernetes
+cluster using their Auth0 credentials.
+
+## Prerequisites
+
+This how-to guide assumes that you have already [installed the Pinniped Supervisor]({{< ref "install-supervisor" >}}) with working ingress,
+and that you have [configured a FederationDomain to issue tokens for your downstream clusters]({{< ref "configure-supervisor" >}}).
+
+## Create an Auth0 Application
+
+Follow the instructions to [create an application](https://auth0.com/docs/get-started/auth0-overview/create-applications).
+
+For example, to create an app:
+
+1. In the [Auth0 Admin Console](https://manage.auth0.com/), navigate to _Applications_ > _Applications_.
+2. Create a new application:
+   1. Click `+ Create Application`.
+   2. Provide a name. For `Choose an application type`, select `Regular Web Applications`.
+   3. Under `Settings`:
+      1. Note the `Client ID` and `Client Secret`, which will be provided later to Pinniped.
+      2. Update `Allowed Callback URLs` with Pinniped's issuer URL, appending `/callback` to the end. (Example: `https://pinniped.issuer.example.com/callback).
+      3. Under `Advanced Settings`, choose `Grant Types` and make sure that `Authorization Code`, `Refresh Token`, and `Password` are selected.
```

Comment:
nit: Could `Advanced Settings` be a 4th bullet in the parent bullet list to reduce the nesting?  I didn't login to Auth0, so if it makes most sense as this nested list then I'm good with that.

Reply:
makes sense, see next push